src/pages/katalog.tsx

```diff
@@ -3,6 +3,45 @@ import '@fortawesome/fontawesome-free/css/all.min.css';
 import { useState } from "react";
 
 
+function Packet(
+    {price, icons, packetColor1, packetColor2, packetType, info, packetDescription}:
+    {price:String; icons: Array<String>; packetColor1: String; packetColor2: String; packetType: String; info: String; packetDescription:Array<String>;}
```

Comment:
these should be `string` instead of `String` and `string[]` instead of `Array<String>`